src/ui/wxWidgets/wxutils.cpp

```diff
@@ -204,9 +205,15 @@ int pless(int* first, int* second) { return *first - *second; }
 bool IsTaskBarIconAvailable()
 {
 #if defined(__WXGTK__)
-  const wxLinuxDistributionInfo ldi = wxGetLinuxDistributionInfo();
-  if (ldi.Id.IsEmpty() || ldi.Id == wxT("Ubuntu") || ldi.Id == wxT("Fedora"))
-    return false;
+  const wxVersionInfo verInfo = wxGetLibraryVersionInfo();
+  int major = verInfo.GetMajor();
+  int minor = verInfo.GetMinor();
+  int micro = verInfo.GetMicro();
+  if (major < 3 || (minor == 0 && micro < 4) || (minor == 1 && micro < 1)) {
```

Comment:
This check will also match e.g. "4.0.4". Is it possible to instead get the version as string and compare lexicographically?

Reply:
In general, string version comparison will work incorrectly if any of parts will be more or equal to `10`, it's better to rewrite condition as 
`(major < 3 || (major == 3 && ((minor == 0 && micro < 4) || (minor == 1 && micro < 1))) )` 
(4.x whenever it'll be, shouldn't, have this bug) or more simple (as we don't use 2.x for some time) 
`(major == 3 && ((minor == 0 && micro < 4) || (minor == 1 && micro < 1)))`

Reply:
I’m not sure if the program would even compile with 2.x, but I’d use your longer check just in case.